modules/wsl-distro.nix

```diff
@@ -77,7 +77,24 @@ in
       extraPackages = mkIf cfg.useWindowsDriver [
         (pkgs.runCommand "wsl-lib" { } ''
           mkdir "$out"
-          ln -s /usr/lib/wsl/lib "$out/lib"
+          ln -s /usr/lib/wsl/lib/libcudadebugger.so.1 "$out/lib"
```

Comment:
```suggestion
          # # we cannot just symlink the lib directory because it breaks merging with other drivers that provide the same directory
          ln -s /usr/lib/wsl/lib/libcudadebugger.so.1 "$out/lib"
```